hibike/devices/YogiBear/LED.cpp

```diff
@@ -19,6 +19,9 @@ void ctrl_RED() {
 	} else {
 		digitalWrite(LED_RED, LOW);
 	}
+	if(!isMotorEnabled()) {
```

Comment:
i recommend "factoring out" the digitalWrite call. the if statements set a variable, and at the very end of the function you make a single digitalWrite call. 

Consider the case: 
readVel() > 0, motorEnabled is false. 
Every time this function gets called, the LED will flick slightly as it gets set from HIGH to LOW then back to HIGH